### PR TITLE
Changelog for Hawaii release

### PR DESCRIPTION
**Pull request by kumarashit**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
 /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
Changelog for hawaii release

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Changelog since Greendlan(v1.1.0) release
```
Comment (anvithks) [inline CHANGELOG/CHANGELOG-v1.2.0.md:1]: Typo - > Greenland

**Review by anvithks — CHANGES_REQUESTED** [review]

File: CHANGELOG/CHANGELOG-v1.2.0.md (hunk 1 above)
Comment (kumarashit) [reply CHANGELOG/CHANGELOG-v1.2.0.md]: Fixed

**Review by skdwriting — APPROVED** [review]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**